### PR TITLE
Adding function "bulkUpdate"

### PR DESCRIPTION
**Pull request by ctoesca**
https://docs.arangodb.com/3.0/HTTP/Document/WorkingWithDocuments.html#update-documents

### CODE REVIEW

File: src/collection.js [hunk 1]
```diff
@@ -218,6 +218,25 @@ class BaseCollection {
     return promise
   }
 
+  bulkUpdate ( newValues, opts, cb) {
```
Comment (pluma) [inline src/collection.js]: There's a stray space before `newValues` in the parameter list here.

File: src/collection.js [hunk 2]
```diff
@@ -218,6 +218,25 @@ class BaseCollection {
...
+      cb = opts
+      opts = undefined
+    }
+    if (typeof opts === 'string') {
+      opts = {rev: opts}
+    }
+    const {promise, callback} = this._connection.promisify(cb)
+     
+    this._api.patch(
+      '/document/'+this.name,
```
Comment (pluma) [inline src/collection.js]: This should use a template string to remain consistent with the surrounding code (compare with line 169):

    `/document/${this.name}`,

**Comment by pluma** [comment]
Thank you for the PR. For legal reasons we need a signed Contributor License Agreement (CLA) before we can merge your code into the project.

You can download the form from our website: https://www.arangodb.com/documents/cla.pdf

Once you've filled out and signed it, please send it to us via e-mail: cla@arangodb.com or fax: +49-221-2722999-88.

**Comment by ctoesca** [comment]
I have added 'collection.bulkUpdate' in README.md, ,under Manipulating documents, after 'collection.update' (like in the Arango HTTP API doc).

File: README.md [hunk 3]
```diff
@@ -2654,6 +2655,72 @@ collection.save(doc)
...
+For more information on the *opts* object, see [the HTTP API documentation for working with documents](https://docs.arangodb.com/latest/HTTP/Document/WorkingWithDocuments.html).
+
+**Examples**
+
+```js
+var db = require('arangojs')();
+var collection = db.collection('services');
+var doc1 = {number: 1, hello: 'world1'};
+collection.save(doc1)
+.then(doc1saved => {
```
Comment (ctoesca) [inline README.md]: test

**Comment by pluma** [comment]
Thanks for signing the CLA, and thanks again for the PR. :smile:

**Pull request merged**